src/System.EventLog/src/Configurations.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
+      netcoreapp;
```

Comment:
Please change `netcoreapp` to `netcoreapp-Windows_NT` because this code is all Windows specific.
Unix will get the `netstandard` one.

Reply:
Done